view/src/app/components/Footer/index.tsx

```diff
@@ -0,0 +1,29 @@
+'use client'
+
+import { useState } from 'react'
+
+import "./style.css"
+import { Song } from '@/app/interfaces/Song';
+
+
+interface FooterProps {
+  song: Song
+}
+
+export default function Footer({ song }: FooterProps) {
+  const [musicPlaying, setMusicPlaying] = useState(false)
+
+  return (
+    <footer className="footer">
+      <h3>{song.nome}</h3>
+      <audio
+        id="song"
+        autoPlay
+        src={`/${song.nome}.mp3`}
```

Comment:
To fix the error on console browser:
![image](https://github.com/arthurboeck/media-player/assets/16245919/5aa57e9d-31fd-45bc-b362-52969059a411)

You can change to current line to this:
```
src={song.nome ? `/${song.nome}.mp3` : undefined}
```